packages/datadog-instrumentations/src/protobufjs.js

```diff
@@ -0,0 +1,165 @@
+const shimmer = require('../../datadog-shimmer')
+const { channel, addHook, AsyncResource } = require('./helpers/instrument')
+
+const startSerializeCh = channel('datadog:protobuf:serialize:start')
+const finishSerializeCh = channel('datadog:protobuf:serialize:finish')
+const startDeserializeCh = channel('datadog:protobuf:deserialize:start')
+const finishDeserializeCh = channel('datadog:protobuf:deserialize:finish')
```

Comment:
You should switch to using `TracingChannel`. That way you can avoid _most_ of the code in this file.

Reply:
Done! LMK if I messed anything up